### DISCUSSION TITLE
盒盒盒喝

### DISCUSSION THREAD

**Discussion by Devilsteep** [share121/inter-knot#4668] [其他]
![mmexport1667806891787](https://github.com/user-attachments/assets/7ebb78fc-fe69-4924-97e9-2596070a26e2)
![mmexport1715944046375](https://github.com/user-attachments/assets/ff63a778-257d-465e-be1b-6336d66f5266)

**Comment by JulyLin555** [comment]
？

**Comment by konghuan7759** [comment]
这个东西不能发啊，侵犯肖像权了

**Comment by IXfromThorne** [comment]
？

**Comment by riichi-ippatsu-tsumo** [comment]
你咋不把你自己大头照也发上来呢
